static/app/components/forms/index.stories.tsx

```diff
@@ -0,0 +1,129 @@
+import storyBook from 'sentry/stories/storyBook';
+import useProjects from 'sentry/utils/useProjects';
+
+import BlankField from './fields/blankField';
+import BooleanField from './fields/booleanField';
+import CheckboxField from './fields/checkboxField';
+import DateTimeField from './fields/dateTimeField';
+import EmailField from './fields/emailField';
+import FileField from './fields/fileField';
+import HiddenField from './fields/hiddenField';
+import NumberField from './fields/numberField';
+import RadioField from './fields/radioField';
+import RangeField from './fields/rangeField';
+import SecretField from './fields/secretField';
+import SelectField from './fields/selectField';
+import SentryMemberTeamSelectorField from './fields/sentryMemberTeamSelectorField';
+import SentryProjectSelectorField from './fields/sentryProjectSelectorField';
+import SeparatorField from './fields/separatorField';
+import TextareaField from './fields/textareaField';
+import TextField from './fields/textField';
+import Form from './form';
```

Comment:
Lets make stories for each input field type on its own. A `<Form>` is not the same as a field and having everything in a giant file will not make it easier for people to find things in the future, or iterate on the examples

Reply:
For this I just want to see them all together.

I'm not going to spend more time on this now.

Reply:
IMO this is a very good starting point for seeing "what can I do with a form"

Reply:
I'd rename it `static/app/components/forms/fields/index.stories.tsx` at the very least. It's really not showing anything to do with `<Form>` specifically.

Reply:
Let me have it render the form state real quick, then it will be more like a form

Reply:
The "default" example for `<Form>` probably could just be without the fields, then it'll be easier to see the Submit button gets auto-inserted.

Then another example with fields and state is helpful

Reply:
I'll move it over. Will add the form state thing later